docs/releases/5.2.txt

```diff
@@ -123,7 +123,12 @@ CSRF
 Database backends
 ~~~~~~~~~~~~~~~~~
 
-* ...
+* Django now supports `oracledb`_ version 2.3.0 or higher.
```

Comment:
We should move this to the "Miscellaneous" backwards incompatible changes section

```
* The minimum supported version of `oracledb`_ is increased from 1.3.2 to
  2.3.0.
```

Reply:
Updated. Thanks.